skia-safe/src/modules/svg.rs

```diff
@@ -208,28 +208,8 @@ fn decode_base64(value: &str) -> Vec<u8> {
         .chars()
         .filter(|&c| !is_html_space(c))
         .collect::<String>();
-    let mut input = &*without_spaces;
 
-    if input.len() % 4 == 0 {
-        if input.ends_with("==") {
-            input = &input[..input.len() - 2]
-        } else if input.ends_with('=') {
-            input = &input[..input.len() - 1]
-        }
-    }
-
-    if input.len() % 4 == 1 {
-        return Vec::new();
-    }
-
-    if input
-        .chars()
-        .any(|c| c != '+' && c != '/' && !c.is_alphanumeric())
-    {
-        return Vec::new();
-    }
```

Comment:
The same thing is checked by the `base64` crate [here](https://github.com/marshallpierce/rust-base64/blob/7f81bafe0e76bcdb84f3069d81d56d77a7a24280/src/engine/general_purpose/decode.rs#L77-L79).